gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# fix double click bug for swiping

### Description

Only 1 click is required to update the stack at `/<eid>/match`.

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
```

1.  Mark some users as interested in / going to the same event.
1.  Navigate to `/<eid>/match`. Click "Thumbs Up" or "Thumbs Down". See that the user disappears from the stack after the first click. Verify in `/admin` that the DB updates accordingly. Once #145 is accepted, a popup will appear on matches as well (I tested this but decided to keep the PRs separate).

### Miscellaneous Comments

*   Basically the issue was that the `users` variable was not being updated during a POST.

### Checklist
